packages/core/integration-tests/test/integration/babel-config-js-multitarget-transform-runtime/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  "presets": ["@parcel/babel-preset-env"],
+  "plugins": ["@parcel/transform-runtime"]
```

Comment:
Maybe `@parcel/babel-plugin-transform-runtime` to be consistent with `@parcel/babel-preset-env`?